test/abstracts.md

```diff
@@ -0,0 +1,107 @@
+# Abstract
+
+The following content represents ideal steps
+performed by hyperHTML to grant performance,
+recycle everything it can, and update layouts.
+
+It's rather a specification than current implementation,
+but the goal is to optimize such specification as much as possible
+and achieve best performance on top of that.
+
+```
+hyperHTML(statics, ...interpolations)
+│
+└▶  is `statics` known?
+    │
+    ├▶  YES
+    │
+    │   is current `context` representing `statics` ?
```

Comment:
Same with context. Then it's easier to make a mental model of how statics and a given context relate to each other.